conbench/app/_plots.py

```diff
@@ -150,12 +150,18 @@ def simple_bar_plot(benchmarks, height=400, width=400, vbar_width=0.7):
     return p
 
 
-def _should_format(data, unit):
+# Using `Optional[float]` here only because of BenchmarkResult.mean being
+# nullable as of today.
+def _should_format(floats: list[Optional[float]], unit):
```

Comment:
I'm still a little worried about this 3.9 style of type hints (i.e. `list[X]` instead of `List[X]`). I know `conbench.app._plots` will probably not be imported by the `conbench` CLI, but it's hard to know for sure since we're not running tests using python 3.8 right now. See #616

Reply:
Thanks for connecting dots. Did this once again unintentionally. I will of course try to keep this in mind, but we have to implement a structural change if we want to prevent this from happening.